app/views/admin/delete/owners-confirm.njk

```diff
@@ -0,0 +1,52 @@
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% extends '_layout.njk' %}
+
+{% block pageTitle %}{{ serviceName }}
+{% endblock %}
+
+{% block beforeContent %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+      {{ govukBackLink({
+        text: "Back",
+        href: model.backLink,
+        attributes: {
+          id: "back"
+        }
+        })
+      }}
+    </div>
+  </div>
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <form class="form" method="post">
+      <h1 class="govuk-heading-l">
+        You are about to delete {{ model.selectedList.length }} dog owner {% if model.selectedList | length == 1 %}record {% else %}records{% endif %}
+      </h1>
+      <p class="govuk-body">Deleted records no longer appear in search results.</p>
+      <p class="govuk-body">You have 90 days to raise a support ticket to recover a deleted dog owner record.</p>
+      {% for ownerReference in model.selectedList %}
```

Comment:
I notice you're posting the payload (data retrieved from the session). I just grab the data from the session rather than the extra network traffic. Not an issue, just a difference in the mechanisms so not consistent. Happy for you to leave as is though

Reply:
wonder what would happen if the session expired at this point